app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlAdapter.kt

```diff
@@ -299,6 +299,8 @@ class SessionControlAdapter(
             RecentBookmarksHeaderViewHolder.LAYOUT_ID -> RecentBookmarksHeaderViewHolder(view, interactor)
             RecentVisitsHeaderViewHolder.LAYOUT_ID -> RecentVisitsHeaderViewHolder(
                 view,
+                composeView = ComposeView(parent.context),
+                viewLifecycleOwner,
                 interactor
             )
```

Comment:
We don't need to render the inflate xml layout anymore. Can you follow the changes in https://github.com/mozilla-mobile/fenix/pull/22039/commits/7a897670b3ddb0107539a69be503d3c1f4508890#diff-83c390c3c57ac8e0af0fe047dc202020e7cb179ffdcb1ab563ba658aff0e386a